widgets/google-map/tpl/static-map.php

```diff
@@ -1,3 +1,9 @@
+<?php
+if( !empty( $destination_url ) ) echo '<a href="' . $destination_url . '" ' . ( $new_window ? 'target="_blank"' : '' ) . '>'; ?>
```

Comment:
Hm. So would something more like:

```
<?php if( !empty( $destination_url ) ){ ?>
<a href="<?php echo $destination_url ?>" <?php ( $new_window ? $new_window : '' ); ?>>
<?php } ?> "

```

Or do you want me to try and offload it so it comes "pre-setup" when passed from get_template_variables?